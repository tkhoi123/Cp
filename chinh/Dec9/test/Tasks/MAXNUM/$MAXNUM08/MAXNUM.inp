8
891752 6307 713706 277566 537812 780311 181095 630746 