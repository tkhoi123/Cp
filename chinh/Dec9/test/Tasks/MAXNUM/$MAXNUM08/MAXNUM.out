8917527803117137066307630746537812277566181095